controllers/velero.go

```diff
@@ -448,6 +451,22 @@ func (r *DPAReconciler) customizeVeleroContainer(dpa *oadpv1alpha1.DataProtectio
 	return nil
 }
 
+func getClientBurst(dpa *oadpv1alpha1.DataProtectionApplication) int {
+	if dpa.Spec.Configuration.Velero != nil && dpa.Spec.Configuration.Velero.ClientBurst != nil {
+		return *dpa.Spec.Configuration.Velero.ClientBurst
+	}
+	// Velero default until 1.13
+	return 30
```

Comment:
no need set defaults via DPA, velero controller already does this internally

Reply:
done